zipline/pipeline/factors/factor.py

```diff
@@ -647,12 +647,14 @@ def zscore(self, mask=NotSpecified, groupby=NotSpecified):
         def zscore(row):
             return (row - nanmean(row)) / nanstd(row)
 
-        return GroupedRowTransform(
+        zscore_factor = GroupedRowTransform(
             transform=zscore,
             factor=self,
             mask=mask,
             groupby=groupby,
         )
+        zscore_factor.window_safe = True
+        return zscore_factor
```

Comment:
I added `window_safe` as a class attribute to `Term`, which I think is typically what we want, but in this specific case we only want to overwrite it for an instance of `GroupedRowTransform`, not the class itself.